CerebNet/run_prediction.py

```diff
@@ -94,6 +102,19 @@ def setup_options():
 
 
 def main(args):
+    """
+    Main function to run the inference based on the given command line arguments.
+
+    Parameters
+    ----------
+    args : argparse.Namespace
+        Command line arguments parsed by `argparse.ArgumentParser`.
+
+    Returns
+    -------
```

Comment:
should not return None, int or str -> int= return code; str = error message.
return int=0 --> success

Reply:
This pattern is something I want to implement throughout FastSurfer.
-> main() functions return 0 if successfull, and an error message (str) if unsuccessful.
In general, they should also not Raise Exceptions (but exception objects might be logged).

Reply:
Noted.